dist/js/cookie-compliance-for-wordpress.js

```diff
@@ -131,6 +131,7 @@
       cacheDom: function cacheDom() {
         this.$el = cacheMainElements.$el;
         this.$settingsModal = this.$el.find('#cookie-popup');
+        this.$body = $('body');
```

Comment:
remove bc we have a global body we are storing in DOM

Reply:
Done :D thank you!